### ISSUE TITLE
Images become stretchy vertically  on iPhone Safari

### ISSUE THREAD

**Issue by aksjdjkasdjk** [neptunian/react-photo-gallery#207]
Images become stretchy vertically on iPhone Safari

**Comment by doowhsalf** [comment]
it seems to be the same on Chrome. Using v8.0.0

**Comment by crisanmm** [comment]
I am experiencing the same issue, does the bug have anything to do with iOS/iPadOS 15? I remember not having this issue before updating.

Tested and no issues on macOS 11.4 and Android 11. Wondering if this is caused by the WebKit implementation?

**Comment by aksjdjkasdjk** [comment]
> I am experiencing the same issue, does the bug have anything to do with iOS/iPadOS 15? I remember not having this issue before updating.
> 
> Tested and no issues on macOS 11.4 and Android 11. Wondering if this is caused by the WebKit implementation?

No, it has no link with IOS15. The main reason is that, the parent div of the images have "display:flex;" but there is no "align-items" by default align-items is stretch

**Comment by crisanmm** [comment]
I applied the following style and the issue seems to be gone.

```css
.react-photo-gallery--gallery > div {
  align-items: stretch;
  & > img {
    min-height: 100%;
  }
}
```

Don't know why this works, but it does...

**Comment by aksjdjkasdjk** [comment]
> I applied the following style and the issue seems to be gone.
> 
> ```css
> .react-photo-gallery--gallery > div {
>   align-items: stretch;
>   & > img {
>     min-height: 100%;
>   }
> }
> ```
> 
> Don't know why this works, but it does...

I will try that

**Comment by sergomet** [comment]
Missing width and height props for photos also prevent them from displaying stretchily.
